browser/autocomplete/sources.gni

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import("//brave/components/brave_webtorrent/browser/buildflags/buildflags.gni")
+import("//brave/components/ipfs/buildflags/buildflags.gni")
+import("//extensions/buildflags/buildflags.gni")
+
+brave_browser_autocomplete_sources = [
+  "//brave/browser/autocomplete/brave_autocomplete_scheme_classifier.cc",
+  "//brave/browser/autocomplete/brave_autocomplete_scheme_classifier.h",
+]
+
+brave_browser_autocomplete_deps = [
+  "//base",
+  "//brave/common",
+  "//brave/components/brave_webtorrent/browser/buildflags",
+  "//brave/components/ipfs/buildflags",
+  "//chrome/browser/profiles:profile",
```

Comment:
Wondering whether we actually need this one and similar `chrome` dependencies, given that we inject our sources into the target that has them

Reply:
If it's ok to you, I think it's better to keep them because that way it's clearer what exactly a given subset of sources depend on, which IMHO makes it harder for people to forget about adding new deps as needed when adding new files to these `sources.gni` in the future. WDYT?